packages/client/src/components/groupSettings/alert.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Alert } from 'antd';
+
+export default class GroupSettingsAlert extends React.PureComponent {
+
+  render() {
+    return <Alert
+           message='Only system admins can update these settings'
```

Comment:
title: `These settings can be updated in Admin Portal`
desc: `Please update them in Admin Portal or contact the administrator.`